src/sage/geometry/polyhedron/library.py

```diff
@@ -2226,22 +2242,22 @@ def six_hundred_cell(self, exact=False, backend=None):
 
         EXAMPLES::
 
-            sage: p600 = polytopes.six_hundred_cell()                                                                   # optional - sage.groups
-            sage: p600                                                                                                  # optional - sage.groups
+            sage: p600 = polytopes.six_hundred_cell(); p600                             # optional - sage.groups
             A 4-dimensional polyhedron in RDF^4 defined as the convex hull of 120 vertices
-            sage: p600.f_vector()  # long time ~2sec                                                                    # optional - sage.groups
+            sage: p600.f_vector()  # long time ~2sec                                    # optional - sage.groups
             (1, 120, 720, 1200, 600, 1)
 
         Computation with exact coordinates is currently too long to be useful::
 
-            sage: p600 = polytopes.six_hundred_cell(exact=True) # not tested - very long time                           # optional - sage.groups
-            sage: len(list(p600.bounded_edges()))               # not tested - very long time                           # optional - sage.groups
+            sage: p600 = polytopes.six_hundred_cell(exact=True)  # not tested - very long time, optional - sage.groups
+            sage: len(list(p600.bounded_edges()))                # not tested - very long time, optional - sage.groups
             720
```

Comment:
Now we have two formats `# ... # optional - ...`, `# ..., optional - ...` for the same thing?

I prefer the first format.

Reply:
I'm using the second format here because there were no alignment opportunities.